sdk/communication/Azure.Communication.Administration/tests/PhoneNumberAdministrationClient/PhoneNumberAdministrationClientLiveTests.cs

```diff
@@ -0,0 +1,233 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.WebSockets;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Azure.Communication.Administration.Models;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Communication.Administration.Tests
+{
+    /// <summary>
+    /// The suite of tests for the <see cref="PhoneNumberAdministrationClient"/> class.
+    /// </summary>
+    /// <remarks>
+    /// These tests have a dependency on live Azure services and may incur costs for the associated
+    /// Azure subscription.
+    /// </remarks>
+    public class PhoneNumberAdministrationClientLiveTests : PhoneNumberAdministrationClientLiveTestBase
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PhoneNumberAdministrationClientLiveTests"/> class.
+        /// </summary>
+        /// <param name="isAsync">A flag used by the Azure Core Test Framework to differentiate between tests for asynchronous and synchronous methods.</param>
+        public PhoneNumberAdministrationClientLiveTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [Test]
+        [TestCase(null, TestName = "GetAllSupportedCountries")]
+        [TestCase("en-US", TestName = "GetAllSupportedCountriesEnUsLocale")]
+        public async Task GetAllSupportedCountries(string locale)
```

Comment:
The compiler doesn't catch it but locale should be nullable here.

```suggestion
        public async Task GetAllSupportedCountries(string? locale)
```